geode-core/src/main/java/org/apache/geode/cache/configuration/CacheConfig.java

```diff
@@ -1140,6 +1140,34 @@ public RegionConfig findRegionConfiguration(String regionPath) {
     protected String orderPolicy;
     @XmlAttribute(name = "forward-expiration-destroy")
     protected Boolean forwardExpirationDestroy;
+    @XmlAttribute(name = "pause-event-processing")
+    protected Boolean pauseEventProcessing;
+
+    /**
+     * Gets the value of whether the processing of the events queued is paused or not
```

Comment:
Gets the value of whether the processing of the events queue is created paused or not?

Reply:
Done